conductor_api/src/conductor/broadcaster.rs

```diff
@@ -0,0 +1,36 @@
+use holochain_core_types::{error::HolochainError, json::JsonString};
+use jsonrpc_ws_server::ws;
+
+/// An abstraction which represents the ability to (maybe) send a message to the client
+/// over the existing connection
+#[derive(Debug)]
+pub enum Broadcaster {
+    Ws(ws::Sender),
+    Noop,
+}
+
+impl Drop for Broadcaster {
+    fn drop(&mut self) {
+        match self {
+            Broadcaster::Ws(sender) => sender.close(ws::CloseCode::Normal).unwrap_or(()),
+            Broadcaster::Noop => (),
+        }
+    }
+}
+
+impl Broadcaster {
+    pub fn send<J>(&self, msg: J) -> Result<(), HolochainError>
```

Comment:
code comment on this fn?

Reply:
@Connoropolous it seems a bit redundant since this is the only method on this impl and seems clear that this is the function that "makes it go"